customize

```diff
@@ -31,17 +31,21 @@ set_apt_sources() {
     cat <<EOF >> ${ROOTDIR}/etc/apt/sources.list
 # Repository: $SUITE
 deb $APT_MIRROR $SUITE $COMPONENTS
-deb-src $APT_MIRROR $SUITE $COMPONENTS
```

Comment:
I don't understand the impact of removing this line. Are you removing support for debian-src repository?